### ISSUE TITLE
empty output and empty parallel-weights.json when tests complete

### ISSUE THREAD

**Issue by jcamden** [tnicola/cypress-parallel#17]
Greetings, @tnicola!

I've set up these commands as per the README:
```
"cy:run": "cypress run",
"cy:parallel-local": "cypress-parallel -s cy:run -t 3 -d cypress/integration/test"
```
Relative to package.json, I have three spec files in cypress/integration/test. Running `yarn cy:parallel-local` starts up cypress-parallel. The tests then run invisibly. (I can see the activity hitting my API.) When the tests conclude, this is printed to the console:
```
Thread 0 time: 0s
Thread 1 time: 0s
Thread 2 time: 0s
┌─────────────────────────┬────────┬───────┬─────────┬─────────┬─────────┐
│ Spec                    │ Time   │ Tests │ Passing │ Failing │ Pending │
└─────────────────────────┴────────┴───────┴─────────┴─────────┴─────────┘
Generated file parallel-weights.json.
```
The parallel-weights.json file is just an empty object. I do get an .mp4 generated for each spec, and the results logged to Cypress Dashboard (if use the --record option). However, it seems that the tests are being run in serial and not in parallel. Running with Yarn 2. Let me know if you have any ideas! Thanks!

**Comment by fearhq** [comment]
I had the same problem, figured out it was due to the custom mocha reporter sending out two back-to-back end suite events, causing two messages to be put on the command line. The parent reads these lines as one block, can't decode the json, and thus doesn't add the test results to the table. I've fixed it locally by excluding the event that doesn't have a filename. May submit a PR

**Comment by tnicola** [comment]
Shoud be fixed now!Check out the new version 0.6.0
